androidNode/src/androidMain/kotlin/network/bisq/mobile/android/node/mapping/Mappings.kt

```diff
@@ -1120,6 +1120,7 @@ class Mappings {
 
     object SettingsMapping {
         // toPojo method not implemented as we do not have a settings value object in the domain
+        private const val DEFAULT_DAYS_AFTER_REDACTING_TRADE_DATA = 0
```

Comment:
It is 90 days: See: SettingsService.DEFAULT_NUM_DAYS_AFTER_REDACTING_TRADE_DATA = 90;

Reply:
Also it should use the newly added value not just a default value. On Bisq 2 the new version is in main.